.htaccess

```diff
@@ -1,5 +0,0 @@
-RewriteEngine on
```

Comment:
Is it intended to remove this?

Reply:
No. That's not right. I'll see what's up.